admin/data/sql/migrations/20190221174823.do.add-age-reason-pupil-table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE [mtc_admin].[pupil] ADD ageReason NVARCHAR(1000) NULL
```

Comment:
That is a lot of length on the reason field.  Does it need to be that long?  Do we mind have such long records inside the pupil table?   How often would we access the reason?  

Another pattern we could consider would be a pupilNote table (pupil_id, note_id) that offloaded the longer notes to a separate 'notes' table.

@GuyHarwood - thoughts?

Reply:
@jon-shipley same length is applied on access arrangements reason field and yes the requirement would be to be max 1000 characters long.

Reply:
We could offload it to `pupilAgeReason` table (pupil_id, reason)

Reply:
@GeorgeChatzigiannis please use `nvarchar(max)` https://docs.microsoft.com/en-us/sql/t-sql/data-types/nchar-and-nvarchar-transact-sql?view=sql-server-2017

Reply:
@GeorgeChatzigiannis comment updated as i understand this is against pupil table, which is fairly central.

Reply:
@GuyHarwood @jon-shipley updated as discussed